lib/shared/drive-constraints.js

```diff
@@ -208,3 +208,162 @@ exports.isDriveValid = (drive, image) => {
 exports.isDriveSizeRecommended = (drive, image) => {
   return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'recommendedDriveSize' ], UNKNOWN_SIZE);
 };
+
+/**
+ * @summary Drive/image compatibility status messages.
+ * @public
+ * @type {Object}
+ *
+ * @description
+ * Status messages intended to be shown to the user.
+ */
+exports.COMPATIBILITY_STATUS_MESSAGES = {
```

Comment:
I suggest to keep it as it is for now. The `lib/shared/messages.js` is a mess I'm looking forward to fix soon, given that their structure no longer matches what we're imposing on `lib/shared/errors.js`. I'll send a PR that refactors all these occurrences next week.